src/components/PokemonCard/PokemonCard.tsx

```diff
@@ -32,15 +33,11 @@ export function PokemonCard({ pokemonItem }: Props): JSX.Element {
     getPokemonsInfo();
   }, [pokemonItem.url]);
 
-  if (isLoading) {
-    return <Loader />;
-  } else {
-    return (
-      <div
-        className={isActive ? 'activeCard' : 'pokemon-card'}
-        key={pokemonItem.name}
-        onClick={() => setIsActive(!isActive)}
-      >
+  return (
+    <div className={isPokemonSelected ? 'active-card' : 'pokemon-card'} onClick={onClick}>
```

Comment:
use one class name for common styles and add specific class name for selected item.